spec/tree-view-spec.coffee

```diff
@@ -3007,6 +3007,70 @@ describe "TreeView", ->
         expect(atom.notifications.getNotifications()[0].getMessage()).toContain 'Opening folder in Finder failed'
         expect(atom.notifications.getNotifications()[0].getDetail()).toContain 'ENOENT'
 
+  describe "showCurrentFileInFileManager()", ->
+    it "does nothing when no file is opened", ->
+      expect(atom.workspace.getPaneItems().length).toBe(0)
+      expect(treeView.showCurrentFileInFileManager()).toBeUndefined()
+
+    it "does nothing when only an untitled tab is opened", ->
+      waitsForPromise ->
+        atom.workspace.open()
+      runs ->
+        $(workspaceElement).focus()
+        expect(treeView.showCurrentFileInFileManager()).toBeUndefined()
+
+    describe "showCurrentFileInFileManager() when some file is opened", ->
```

Comment:
As these tests are basically copies of the tests from `showSelectedEntryInFileManager()`, and both use the same underlying `openInFileManager()` code path, I think these tests can be removed.